src/webgpu/api/operation/adapter/requestAdapter.spec.ts

```diff
@@ -114,10 +114,15 @@ g.test('requestAdapter')
       ...(forceFallbackAdapter !== undefined && { forceFallbackAdapter }),
     });
 
-    // failing to create an adapter when forceFallbackAdapter is true is ok.
-    if (forceFallbackAdapter && !adapter) {
-      t.skip('No adapter available');
-      return;
+    assert(typeof adapter?.info.isFallbackAdapter === 'boolean');
```

Comment:
This assert will fail if `adapter` is null.

I adjusted the logic a bit and also switched to `t.expect`. PTAL

Reply:
Thanks for catching! LGTM

FWIW, I would have followed the following pattern.

```js
    if (!adapter) {
      // Failing to create an adapter is only OK when forceFallbackAdapter is true.
      t.expect(forceFallbackAdapter === true);

      // Mark the test as skipped (as long as nothing else failed before this point).
      t.skip('No fallback adapter available');
      return;
    }
    
    t.expect(adapter.info.isFallbackAdapter === Boolean(forceFallbackAdapter));
    await testAdapter(t, adapter);
```

Reply:
That makes sense! Feel free to apply that and land